### PR TITLE
docs: update get-verions section of Azure Stack documentation

### PR DESCRIPTION
**Pull request by haofan-ms**
**Reason for Change**:
Doc was not update when the PR (https://github.com/Azure/aks-engine/pull/3394) was merged, thus making the change.

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [ ] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Review by jackfrancis — DISMISSED** [review]
/lgtm

File: docs/topics/azure-stack.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ If you need to expose more than 5 services, then the recommendation is to route
 
 ### get-versions command
 
-The output of the `get-versions` command only pertains to Azure and not Azure Stack Hub clouds. The different upgrade paths can be found [here](https://docs.microsoft.com/en-us/azure-stack/user/kubernetes-aks-engine-release-notes?view=azs-2005#kubernetes-version-upgrade-path-in-aks-engine-0510).
+By default, `aks-engine get-versions` shows which Kubernetes versions are supported by each AKS Engine release on Azure's public cloud. Include flag `--azure-env` to get the list of supported Kubernetes versions on a custom cloud such as an Azure Stack Hub cloud (`aks-engine get-versions --azure-env AzureStackCloud`). Upgrade paths for Azure Stack Hub can also be found [here](https://docs.microsoft.com/en-us/azure-stack/user/kubernetes-aks-engine-release-notes?view=azs-2005#kubernetes-version-upgrade-path-in-aks-engine).
```
Comment (jadarsie) [inline docs/topics/azure-stack.md:173]: I think `en-us` should be removed from the URL.
I think we should remove `view=azs-####` from the URL
The URL points to a non-existing anchor (`#kubernetes-version-upgrade-path-in-aks-engine`) I would remove it as it looks like the section title changes with every release.

Comment (haofan-ms) [reply docs/topics/azure-stack.md:173]: Thanks Javier, I get your point.I will only keep the constants in the URL and make it sure redirect automatically every time.

**Pull request merged**